R/gen_population.R

```diff
@@ -6,64 +6,131 @@
 #' @param total_pop The overal population count of interest - the default is
 #'   1000 people
 #' @param groups A character vector of groups - the default is set for age
-#' groups: c("0-4","5-14","15-24","25-34","35-44","45+")
+#' groups: c("0-4","5-14","15-29","30-44","45+")
 #' @param strata A character vector for stratifying groups -
 #'   the default is set for gender: c("Male", "Female")
 #' @param proportions A numeric vector specifying the proportions (as decimals)
-#'   for each age group - the default repeats c(0.182, 0.278, 0.26, 0.11, 0.07,
-#'   0.10) for genders.  However you can change this manually, make sure to
-#'   have the length equal to groups times strata.
+#'   for each group - the default repeats c(0.079, 0.134, 0.139, 0.082, 0.067)
+#'   for genders.  However you can change this manually, make sure to
+#'   have the length equal to groups times strata (or half thereof).
+#'   These defaults are based of MSF general emergency intervention standard
+#'   values.
+#' @param counts A numeric vector specifying the counts for each group.
+#'   The default is NULL - as most often proportions above will be used.
+#'   If is not NULL then total_pop and proportions will be ignored.
+#'   Make sure the length of this vector is equal to groups time strata (or if it
+#'   is half then it will repeat for each strata).
+#'   For reference, the MSF general emergency intervention standard values
+#'   are c(7945, 13391, 13861, 8138, 6665) based on above
+#'   groups for a 100,000 person population.
 #' @param tibble Return data as a tidyverse tibble (default is TRUE)
 #' @importFrom dplyr bind_cols
```

Comment:
Can you add an example of how this function is used? 

It would be good to show how adding proportions and counts differs.